drive_to_gps/include/calculate_degrees.h

```diff
@@ -0,0 +1,38 @@
+#ifndef CALCULATE_DEGREES_H
```

Comment:
this should be in `include/utils/utils.h` or smth similar. source code should also be in the src instead of directly in the header.